tasks.rb

```diff
@@ -57,6 +57,7 @@ def create_hpxmls
     'invalid_files/water-heater-location.xml' => 'base.xml',
     'invalid_files/water-heater-location-other.xml' => 'base.xml',
     'invalid_files/slab-zero-exposed-perimeter.xml' => 'base.xml',
+    'invalid_files/only-wall-tiny.xml' => 'base-atticroof-flat.xml',
```

Comment:
Remove this, we'll cover this and more in the PR to ensure spaces have minimum adjacent surface types.